README.md

```diff
@@ -8,22 +8,12 @@ Fleet is the lightweight, open source telemetry platform for servers and worksta
 
 ## Try Fleet
 
-#### With [Node.js](https://nodejs.org/en/download/) and [Docker](https://docs.docker.com/get-docker/) installed:
-
-```bash
-# Install the Fleet command-line tool
-sudo npm install -g fleetctl
-# Run a local demo of the Fleet server
-sudo fleetctl preview
-```
-
-> Windows users can omit `sudo`, and should run the command in `Cmd`/`PowerShell` as administrators.
-
-The Fleet UI is now available at http://localhost:1337.
+Head to [fleetdm.com/get-started](https://fleetdm.com/get-started) to play in Fleet Sandbox.
```

Comment:
Do we need more explanation of what Fleet Sandbox is? "One-off cloud instance for quickly trying out fleet"

Reply:
@zhumo this is a great idea. I added updated the sentences to the following:

> Head to [fleetdm.com/try-fleet](https://fleetdm.com/try-fleet/register) to fire up a one-off cloud instance for quickly trying out Fleet.